docs/pages/eas-update/introduction.mdx

```diff
@@ -15,7 +15,25 @@ All apps running the `expo-updates` library have the ability to receive updates.
 
 Still using Classic Updates? We moved those docs to [the archive](/archive/classic-updates/introduction).
 
-### Get started
+## Pitch
+
+#### Improve User Experience
```

Comment:
```suggestion
### Improve User Experience
```

Let's not skip heading levels (h2 to h4) as per our writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings